src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -116,20 +189,45 @@ public static void main(String[] args) throws Exception {
             t.printStackTrace();
             System.exit(1);
         }
+    }
+
+    private static void executePackageStep(SubstrateDispatcher dispatcher) {
+        System.err.println("Packaging...");
+        try {
+            dispatcher.nativePackage();
+        } catch (Throwable t) {
+            System.err.println("Packaging failed with an exception");
```

Comment:
print t as well (t.getMessage() is not shown in stacktraces, but may contain relevant info)

Reply:
The full stack trace is printed the line below, which also prints the message exception.